central/scannerdefinitions/handler/handler.go

```diff
@@ -109,6 +119,14 @@ func (h *httpHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *httpHandler) get(w http.ResponseWriter, r *http.Request) {
+	fileType := r.URL.Query().Get("type")
```

Comment:
what does `type` mean?

Reply:
that's parameter for getting container-name-repos-map.json or  repository-to-cpe.json